scripts/core/exp-usm-import-release.yml

```diff
@@ -30,10 +30,10 @@ name: ReleaseExp
 details:
     - "Release memory from USM"
 params:
-    - type: $x_context_handle_t
+    - type: "const $x_context_handle_t"
```

Comment:
Also don't think this should be `const` qualified, see above.

Reply:
OK

Reply:
I'm closing this PR because I don't think these changes are needed. There is a remapping, including casting, between SYCL objects and UR objects anyway.